### ISSUE TITLE
Unable to use slither after installing slitherin

### ISSUE THREAD

**Issue by vishnuram1999** [pessimistic-io/slitherin#93]
After installing the slitherin and I tried running following command in one of my smart contracts
```
slither contracts/<Contract-Name> --config slither.config.json
```
I got the following error
```
Traceback (most recent call last):
  File "/home/linuxbrew/.linuxbrew/opt/python@3.11/lib/python3.11/site-packages/slither/__main__.py", line 810, in main_impl
    ) = process_all(filename, args, detector_classes, printer_classes)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/linuxbrew/.linuxbrew/opt/python@3.11/lib/python3.11/site-packages/slither/__main__.py", line 101, in process_all
    ) = process_single(compilation, args, detector_classes, printer_classes)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/linuxbrew/.linuxbrew/opt/python@3.11/lib/python3.11/site-packages/slither/__main__.py", line 81, in process_single
    return _process(slither, detector_classes, printer_classes)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/linuxbrew/.linuxbrew/opt/python@3.11/lib/python3.11/site-packages/slither/__main__.py", line 120, in _process
    slither.register_detector(detector_cls)
  File "/home/linuxbrew/.linuxbrew/opt/python@3.11/lib/python3.11/site-packages/slither/slither.py", line 188, in register_detector
    instance = detector_class(compilation_unit, self, logger_detector)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/linuxbrew/.linuxbrew/opt/python@3.11/lib/python3.11/site-packages/slither_pess/detectors/uni_v2.py", line 332, in __init__
    fileJson = open(full_path)
               ^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/home/linuxbrew/.linuxbrew/opt/python@3.11/lib/python3.11/site-packages/slither_pess/detectors/../utils/deflat_tokens.json'
ERROR:root:Error in contracts/WithdrawContract.sol
ERROR:root:Traceback (most recent call last):
  File "/home/linuxbrew/.linuxbrew/opt/python@3.11/lib/python3.11/site-packages/slither/__main__.py", line 810, in main_impl
    ) = process_all(filename, args, detector_classes, printer_classes)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/linuxbrew/.linuxbrew/opt/python@3.11/lib/python3.11/site-packages/slither/__main__.py", line 101, in process_all
    ) = process_single(compilation, args, detector_classes, printer_classes)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/linuxbrew/.linuxbrew/opt/python@3.11/lib/python3.11/site-packages/slither/__main__.py", line 81, in process_single
    return _process(slither, detector_classes, printer_classes)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/linuxbrew/.linuxbrew/opt/python@3.11/lib/python3.11/site-packages/slither/__main__.py", line 120, in _process
    slither.register_detector(detector_cls)
  File "/home/linuxbrew/.linuxbrew/opt/python@3.11/lib/python3.11/site-packages/slither/slither.py", line 188, in register_detector
    instance = detector_class(compilation_unit, self, logger_detector)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/linuxbrew/.linuxbrew/opt/python@3.11/lib/python3.11/site-packages/slither_pess/detectors/uni_v2.py", line 332, in __init__
    fileJson = open(full_path)
               ^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/home/linuxbrew/.linuxbrew/opt/python@3.11/lib/python3.11/site-packages/slither_pess/detectors/../utils/deflat_tokens.json'
```
Can someone help me with this?

**Comment by Yhtiyar** [comment]
Hi @vishnuram1999, the problem is that when we are building the package, one important json file for one of our detectors is not included. As for now, the quick fix would be ignoring that detector with passing `--exclude pess-uni-v2`   

We are already preparing fix and will release this week.

**Comment by vishnuram1999** [comment]
Thank you, I will use this fix for now.

**Comment by ndkirillov** [comment]
Fixed in #97
